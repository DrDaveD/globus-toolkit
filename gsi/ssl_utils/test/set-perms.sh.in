#!/bin/sh
######################################################################
#
# set-perms.sh
#
# Set the correct permissions on the client and server keys so that
# the GSSAPI/library will accept them.
#
######################################################################

######################################################################
#
# Where to find various certificates
#
SERVER_CERTDIR=@SERVER_CERTDIR@
CLIENT_CERTDIR=@CLIENT_CERTDIR@
CA_CERTDIR=@CA_CERTDIR@

######################################################################
#
# Main Code
#

SERVER_KEY=${SERVER_CERTDIR}/key.pem
CLIENT_KEY=${CLIENT_CERTDIR}/key.pem

echo "Setting permissions on server key to 0400"
chmod 0400 ${SERVER_KEY}

echo "Setting permissions on client key to 0400"
chmod 0400 ${CLIENT_KEY}

echo "Success"
exit 0;

#
# End code
#
######################################################################

