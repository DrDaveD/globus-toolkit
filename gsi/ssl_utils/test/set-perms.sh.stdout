Setting permissions on server key to 0400
Setting permissions on client key to 0400
Success
